examples/code/imperative-programming/file2.topscript

```diff
@@ -14,23 +14,38 @@ let sum_file filename =
     List.fold ~init:0 ~f:(+) numbers)
     ~finally:(fun () -> In_channel.close file)
 ;;
```

Comment:
Should we maybe just add open Stdio at the top of this? We could do it in a subfeature, but it's such an easy fix, and this way we can't forget to fix it...